doc/playbook.rst

```diff
@@ -79,66 +87,71 @@ From the ``main`` branch you can release a new major version (``X.0.0``), a new
 Release tags are semantic version identifiers following the `pep 440 <https://peps.python.org/pep-0440/>`_ specification.
 Please note that the tag does not include a ``v`` at the beginning.
 
-1. Choose Version Number
+#. Choosing the Version Number
+
+   The first step in making a release is choosing an appropriate version number to describe it. This subsection offers guidance on this choice, as well as on differentiating between major and minor releases.
+
+   **To see which changes have occurred since the last release:**
 
-On GitHub.com, navigate to the main page of the repository.
-To the right of the list of files, click the latest release.
-At the top of the page, click **## commits to main since this release**.
-(If there's no such link or it doesn't mention ``main``, the release is probably based off a branch; go up to Releases and try older versions until you find one.)
-This is the list of internal changes that will be included in the next release.
+   On GitHub.com, navigate to the main page of the repository.
+   To the right of the list of files, click the latest release.
+   At the top of the page, click **## commits to main since this release**.
+   (If there's no such link or it doesn't mention ``main``, the release is probably based off a branch; go up to Releases and try older versions until you find one.)
+   This is the list of internal changes that will be included in the next release.
 
-If you are planning to update the Science Pipelines tag, you should also check the `Science Pipelines changelog <https://lsst-dm.github.io/lsst_git_changelog/weekly/>`_.
-In practice, almost any Science Pipelines update is at least a minor version, because new features are added constantly.
-In the future, there may be "patched weekly" builds, which would justify a patch version of Prompt Processing.
+   If you are planning to update the Science Pipelines tag, you should also check the `Science Pipelines changelog <https://lsst-dm.github.io/lsst_git_changelog/weekly/>`_.
+   In practice, almost any Science Pipelines update is at least a minor version, because new features are added constantly.
+   You may need to do a "Quick-Stack" build, or build the science pipelines manually. This would justify a patch version of Prompt Processing.
 
-For the ``prompt_processing`` service, a new major version is triggered by any of the following:
+   **The following changes will always trigger a major release:**
```

Comment:
Maybe remove the "always"? The list is supposed to be exhaustive, and the current wording makes it sound like there are other cases, too.